teamwork/templates/project_details.html

```diff
@@ -21,124 +29,145 @@
-    <div class="col-md-6">
-
-      <!-- Task list -->
-      <h3>Tasks List</h3>
-      <ul class="list-group">
-        {% for task in project.task_set.all %}
-        <li class="list-group-item">
-          <input type="checkbox" 
-          {% if task.done %}
-            checked
-          {% endif %}> {{ task.name }}</input>
-        </li>
-        {% endfor %}
-      </ul>
-      <button type="button" class="btn btn-default" onclick="openEventCreator()">Add task</button>
-    </div>
-    <div class="col-md-6">
-
-      <!-- Deadline -->
-      <h3>Time till deadlines</h3>
-      <ul class="list-group">
-        {% for deadline in deadlines|dictsort:'date' %}
-        <li class="list-group-item">
-          <p><span class="timeUntil">{{ deadline.date|timeuntil }} left</span> <strong>{{ deadline.name }}</strong> ({{ deadline.date|date:"D, d M Y" }}, {{ deadline.date|time:"H:i" }}
-          {% if deadline.location != "" %}, 
+  <button id="add-member" type="button" class="btn btn-default"
+  data-toggle="modal" data-target="#add-member-modal">
+  Add member
+</button>
+{% endif %}
+
+{% include 'modals/add_member.html' %}
+
+<div class="row">
+  <div class="col-md-6">
+
+    <!-- Task list -->
+    <h3>Tasks List</h3>
+    <ul class="list-group">
+      {% for task in project.task_set.all %}
+      <li class="list-group-item">
+        <input type="checkbox"
+        {% if task.done %}
+        checked
+        {% endif %}> {{ task.name }}</input>
+      </li>
+      {% endfor %}
+    </ul>
+    <button type="button" class="btn btn-default" onclick="openEventCreator()">Add task</button>
+  </div>
+  <div class="col-md-6">
+
+    <!-- Deadline -->
+    <h3>Time till deadlines</h3>
+    <ul class="list-group">
+      {% for deadline in deadlines|dictsort:'date' %}
+      <li class="list-group-item">
+        <p><span class="timeUntil">{{ deadline.date|timeuntil }} left</span> <strong>{{ deadline.name }}</strong> ({{ deadline.date|date:"D, d M Y" }}, {{ deadline.date|time:"H:i" }}
+          {% if deadline.location != "" %},
           <i class="glyphicon glyphicon-map-marker"></i> {{ deadline.location }}
           {% endif %})
-          </p>
-        </li>
-        {% empty %}
-        <li class="list-group-item">
-          <p>There is no deadline coming up</p>
-        </li>
-        {% endfor %}
-      </ul>
-    </div>
+        </p>
+      </li>
+      {% empty %}
+      <li class="list-group-item">
+        <p>There is no deadline coming up</p>
+      </li>
+      {% endfor %}
+    </ul>
   </div>
+</div>
 
-  <!-- Time Spent -->
-  <h3>Time Spent</h3>
-  <div class="row">
-
-    <!--  Time spent by each project's members -->
-    <div class="col-md-6">
-      <div id="piechart"></div>
-      <button type="button" class="btn btn-primary" onclick="openEventCreator()">Log time</button>
+<!-- Time Spent -->
+<h3>Time Spent</h3>
+<div class="form-group">
+  <form action="log_time" method="post" id="logTimeForm" class="form-inline">

```

Comment:
Not that important but html is case insensitive so `logTimeForm` is same as `logtimeform`. That's why it's common to use dashes to separate words, like this `log-time-form`.